ElementX/Sources/Screens/ManageAuthorizedSpacesScreen/View/ManageAuthorizedSpacesScreen.swift

```diff
@@ -0,0 +1,117 @@
+//
+// Copyright 2025 Element Creations Ltd.
+//
+// SPDX-License-Identifier: AGPL-3.0-only OR LicenseRef-Element-Commercial.
+// Please see LICENSE files in the repository root for full details.
+//
+
+import Compound
+import SwiftUI
+
+struct ManageAuthorizedSpacesScreen: View {
+    @Bindable var context: ManageAuthorizedSpacesScreenViewModel.Context
+    
+    var body: some View {
+        Form {
+            header
+            if !context.viewState.authorizedSpacesSelection.joinedParentSpaces.isEmpty {
+                joinedParentsSection
+            }
+            if !context.viewState.authorizedSpacesSelection.unknownSpacesIDs.isEmpty {
+                unkwnownSpacesSection
+            }
```

Comment:
Maybe add a bit of whitespace so its easier to read :)
```suggestion
            header
            
            if !context.viewState.authorizedSpacesSelection.joinedParentSpaces.isEmpty {
                joinedParentsSection
            }
            
            if !context.viewState.authorizedSpacesSelection.unknownSpacesIDs.isEmpty {
                unkwnownSpacesSection
            }
```